contracts/strategies/StrategyCurve3CrvVoterProxy.sol

```diff
@@ -0,0 +1,188 @@
+pragma solidity ^0.5.17;
+
+import "@openzeppelinV2/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelinV2/contracts/math/SafeMath.sol";
+import "@openzeppelinV2/contracts/utils/Address.sol";
+import "@openzeppelinV2/contracts/token/ERC20/SafeERC20.sol";
+
+import "../../interfaces/yearn/IController.sol";
+import "../../interfaces/curve/Gauge.sol";
+import "../../interfaces/curve/Mintr.sol";
+import "../../interfaces/uniswap/Uni.sol";
+import "../../interfaces/curve/Curve.sol";
+import "../../interfaces/yearn/Token.sol";
+import "../../interfaces/yearn/VoterProxy.sol";
+
+contract StrategyCurve3CrvVoterProxy {
+    using SafeERC20 for IERC20;
+    using Address for address;
+    using SafeMath for uint256;
+    
+    address constant public want = address(0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490);
+    address constant public crv = address(0xD533a949740bb3306d119CC777fa900bA034cd52);
+    address constant public uni = address(0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D);
+    address constant public weth = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2); // used for crv <> weth <> dai route
+    
+    address constant public dai = address(0x6B175474E89094C44Da98b954EedeAC495271d0F);
+    address constant public curve = address(0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7);
+
+    address constant public gauge = address(0xbFcF63294aD7105dEa65aA58F8AE5BE2D9d0952A);
+    address public constant proxy = address(0x7A99923aA2efa71178BB11294349EC1F6b23a814);
```

Comment:
This will need to be updated once new proxy is deployed

Reply:
Can we add a method `setProxy` (only goverance)?